airbyte-commons-worker/src/test/java/io/airbyte/workers/general/DefaultReplicationWorkerTest.java

```diff
@@ -166,7 +175,9 @@ void test() throws Exception {
         messageTracker,
         recordSchemaValidator,
         workerMetricReporter,
-        connectorConfigUpdater, false);
+        connectorConfigUpdater,
```

Comment:
OPT: this constructor feels very redundant, is it worth abstracting to ease c/p?

Reply:
Done